src/topology/algebra/uniform_group.lean

```diff
@@ -132,6 +132,12 @@ begin
   exact tendsto.comp h tendsto_comap
 end
 
+lemma uniform_continuous_add_group_hom_of_continuous_at_zero
```

Comment:
```suggestion
lemma add_monoid_hom.uniform_continuous_of_continuous_at_zero
```
This allows to use the dot notation as in `f.uniform_continuous_of_continuous_at_zero (hf : continuous_at f 0)`

Reply:
Nice! Thanks for this suggestion, amended.